docs/credential-storage.md

```diff
@@ -29,8 +29,9 @@ For the package to know which model you want to use, you will need to call the f
 
 ```php
 use Webfox\Xero\Xero;
+use App\Models\User;
 
-Xero::useModelStore(Settings::class);
+Xero::useModelStore(User::find(1));
```

Comment:
While the docs are wrong here since it requires an instance of the model, I believe we should still use the fictional Settings model given above we say
> this is useful for applications that have a settings model.

Maybe just do `Settings::first()`?

Reply:
Good catch.